src/loaders/ImageBitmapLoader.js

```diff
@@ -35,9 +35,7 @@ ImageBitmapLoader.prototype = Object.assign( Object.create( Loader.prototype ),
 
 	},
 
-	load: function ( url, onLoad, onProgress, onError ) {
-
-		if ( url === undefined ) url = '';
+	load: function ( url = '', onLoad, onProgress, onError ) {
```

Comment:
This does not really make sense. Default parameters should normally be used for optional parameters. `url` is mandatory. 

@mrdoob What is your preference here. To be, this signature looks odd. I would keep the code as it is. Or remove `if ( url === undefined ) url = '';` in general.